setup.py

```diff
@@ -1,4 +1,9 @@
 import setuptools
+
```

Comment:
Why didn't `isort` complain about this? Or `ruff` if you use that? Please remove this empty line and make sure that `isort` is run.